components/profile/my-network-settings/registry-settings.tsx

```diff
@@ -167,10 +171,22 @@ export default function RegistrySettings({ isGovernorRegistry = false }) {
 
     const getBlock = ({ blockNumber }) => blocks.push(blockNumber);
 
-    if (toAdd.length)
+    const findMinAmount = (tokens: Token[], newAddress: string) =>
+      tokens.find(({ address }) => address.toLowerCase() === newAddress.toLowerCase());
+
+    if (toAdd.length){
+      await Promise.all(toAdd.map(async (address) => {
+        const currentToken = findMinAmount(isTransactional ? transactionalTokens : rewardTokens, address)
+        if(currentToken?.minimum !== '0'){
```

Comment:
If the minimum value must not equal 0, then you should validate it when the user is typing the value. Otherwise, the token won't be saved to the database and no feedback will be returned to the user.

Reply:
I created this logic, to reduce unnecessary requests. If he does not enter a minimum value or enters 0, we do not need to make the request. because the token is created by events and the migration inserts the value 0 by default